apps/chat/src/components/Folder/Folder.tsx

```diff
@@ -849,12 +849,20 @@ const Folder = <T extends ConversationInfo | PromptInfo | DialFile>({
               data-qa="folder-name"
             >
               <Tooltip
-                tooltip={t(
-                  getEntityNameError(isNameInvalid, isInvalidPath, isExternal),
-                )}
-                hideTooltip={!isNameOrPathInvalid}
+                tooltip={
+                  currentFolder.name ||
+                  t(
+                    getEntityNameError(
+                      isNameInvalid,
+                      isInvalidPath,
+                      isExternal,
+                    ),
+                  )
+                }
+                hideTooltip={isNameOrPathInvalid}
```

Comment:
Your can't just remove NOT here, because your hide error always!
Pay attention for what this was done.
Just remove this property to show tooltip always

Reply:
Thanks for your comment, I fixed it.